net/Socket.cpp

```diff
@@ -1742,7 +1742,7 @@ std::string getServerString()
     if (sig)
         return "COOLWSD HTTP Server " + Util::getCoolVersion();
 
-    return "COOLWSD HTTP Server";
+    return " ";
```

Comment:
Intentional to return a single space? Why not a blank string?

Reply:
The reporter hinted at Apache's `SecServerSignature " "` setting. I was not sure, whether an empty string would break something.